src/Uno.Templates/Uno.Templates.csproj

```diff
@@ -86,9 +83,7 @@
 		<Copy SourceFiles="@(TemplateFile)" DestinationFiles="$(TemplateContentFolder)/%(RecursiveDir)%(Filename)%(Extension)" SkipUnchangedFiles="false" />
 		<Copy SourceFiles="content/unoapp/GitHub/Install-WindowsSdkISO.cips" DestinationFiles="$(TemplateContentFolder)/unoapp/AzurePipelines/Install-WindowsSdkISO.cips" SkipUnchangedFiles="false" />
 		<Copy SourceFiles="content/unoapp/MyExtensionsApp.1.DataContracts/WeatherForecast.cs" DestinationFiles="$(TemplateContentFolder)/unoapp/MyExtensionsApp.1.Server/WeatherForecast.cs" SkipUnchangedFiles="false" />
-		<Copy SourceFiles="content/unoapp/MyExtensionsApp.1.Windows/Package.appxmanifest" DestinationFiles="$(TemplateContentFolder)/unoapp/MyExtensionsApp.1.Skia.Gtk/Package.appxmanifest" SkipUnchangedFiles="false" />
-		<Copy SourceFiles="content/unoapp/MyExtensionsApp.1.Windows/Package.appxmanifest" DestinationFiles="$(TemplateContentFolder)/unoapp/MyExtensionsApp.1.Skia.Linux.FrameBuffer/Package.appxmanifest" SkipUnchangedFiles="false" />
-		<Copy SourceFiles="content/unoapp/MyExtensionsApp.1.Windows/Package.appxmanifest" DestinationFiles="$(TemplateContentFolder)/unoapp/MyExtensionsApp.1.Skia.WPF/Package.appxmanifest" SkipUnchangedFiles="false" />
+		<Copy SourceFiles="content/unoapp/MyExtensionsApp.1/Platforms/Windows/Package.appxmanifest" DestinationFiles="$(TemplateContentFolder)/unoapp/MyExtensionsApp.1/Platforms/Skia/Package.appxmanifest" SkipUnchangedFiles="false" />
```

Comment:
Do we still need a skia version of the Package.appxmanifest? Why can't we have a single version of this instead of it being in platform folder?